### PR TITLE
Session details and agenda update

### PR DESCRIPTION
**Pull request by harisood**
Adding detail to the agenda to circulate with the community

### CODE REVIEW

File: docs/events/wg_workshops/2023-12-05-december-meeting/workshop-tre-c4-architecture.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Data architecture of a TRE with C4 modelling language
+
```
Comment (manics) [inline docs/events/wg_workshops/2023-12-05-december-meeting/workshop-tre-c4-architecture.md:2]: Who's running this workshop?

File: docs/events/wg_workshops/2023-12-05-december-meeting/workshop-satre.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# SATRE next steps
+
```
Comment (manics) [inline docs/events/wg_workshops/2023-12-05-december-meeting/workshop-satre.md:2]: Who's running this? Chris Cole?

File: docs/events/wg_workshops/2023-12-05-december-meeting/discussion-multi-tre.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
+# Multi-TRE analysis : can a common governance model breaks data silos?
+
+\*\* **Chair**: Fatemeh Torabi (Swansea University)
```
Comment (manics) [inline docs/events/wg_workshops/2023-12-05-december-meeting/discussion-multi-tre.md:3]:
```suggestion
**Chair**: Fatemeh Torabi (Swansea University)
```

File: docs/events/wg_workshops/2023-12-05-december-meeting/workshop-tre-c4-architecture.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+   - Perform statistical disclosure control
+3. Output trusted analyses
+
+You’d be forgiven for thinking this TRE sounds like a sandbox, but the special ingredient here is the implementation of governance protocols, checking and balancing data uses at key turning points.
+
+To help communicate this approach to TRE design, we have experimented with analogies such as central reference libraries and air traffic control, and will workshop some examples to round off this thought experiment.
+
+## Preparation
+
+TBD
```
Comment (manics) [inline docs/events/wg_workshops/2023-12-05-december-meeting/workshop-tre-c4-architecture.md:25]:
```suggestion
```
There's no time to prepare now 😄

File: docs/events/wg_workshops/2023-12-05-december-meeting/workshop-satre.md (hunk 2 above)
Comment (harisood) [reply docs/events/wg_workshops/2023-12-05-december-meeting/workshop-satre.md:2]: fixed in latest commit

File: docs/events/wg_workshops/2023-12-05-december-meeting/workshop-tre-c4-architecture.md (hunk 4 above)
Comment (harisood) [reply docs/events/wg_workshops/2023-12-05-december-meeting/workshop-tre-c4-architecture.md]: Still putting it in JIC

**Review by Davsarper — APPROVED** [review]
lgtm

**Pull request merged**